src/core/reducers/categories.js

```diff
@@ -67,7 +67,17 @@ export default function categories(state = initialState, action) {
         Object.keys(categoryList).forEach((appName) => {
           Object.keys(categoryList[appName]).forEach((addonType) => {
             categoryList[appName][addonType] = categoryList[appName][addonType]
-              .sort((a, b) => a.name >= b.name)
+              .sort((a, b) => {
+                const x = a.slug;
```

Comment:
These definitions aren't necessary; you can just use `a.slug` directly instead as they'll fit on the lines below and these variable definitions actually obscure the variables rather than explaining their contents.